### DISCUSSION TITLE
Can't find model 'en'.

### DISCUSSION THREAD

**Discussion by Giyn** [explosion/spaCy#7365] [Help: Installation] [answered]
I meet the problem "Can't find model 'en'.". When I use "python3 -m spacy download en" to download en model, nothing happened. I also can not run my codes.

## My Environment

- Operating System: Windows 10
- Python Version Used: 3.7.7
- spaCy Version Used: 2.3.5

**Comment by svlandeg** [comment]
Could you provide the details (commands + log output) of all the commands you ran, starting from the creation of the new env, the installation of spaCy and the download command? 

Just to check whether there were any warnings or anything suspicious, because it's kind of hard for us to help determine what goes wrong otherwise.

**Comment by Giyn** [comment]
Every time I use the command "python -m spacy download en", I get errors:
"Socket.gaierror: [Errno 11004] getaddrinfo failed" and
"Requests.exceptions.ConnectionError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /explosion/spacy-models/master/shortcuts-v2.json (Caused by NewConnectionError(' <urllib3.connection.HTTPSConnection object at 0x00000247E6FEEA08>: Failed to establish a new connection: [Errno 11004] getaddrinfo failed'))”

**Comment by svlandeg** [comment, marked as answer]
Hm, not sure what's causing that, I don't see any issues when I try to download the models myself. Perhaps a firewall on your system or network?

You can try installing the models directly from pip: https://spacy.io/usage/models#download-pip
```
pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.1/en_core_web_sm-2.3.1.tar.gz
```
And then load the model with `spacy.load('en_core_web_sm')`. Hope that works for you!

**Comment by Giyn** [comment]
Great, it works! Thank you very much for your patience!
